sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Change Log azure-data-appconfiguration
+
+## Version 1.0.0-preview.2:
+
+For details on the Azure SDK for Java (August 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview2-java).
+
+- Merged ConfigurationClientBuilder and ConfigurationAsyncClientBuilder into ConfigurationClientBuilder. Method to build each client were added.
+- ConfigurationClientBuilder was made instantiable, static builder method removed from ConfigurationClient and ConfigurationAsyncClient.
+- Builder method credentials renamed to credential and serviceEndpoint to endpoint.
+- Listing operations return PagedFlux and PagedIterable in their respective clients.
+- Non-maximal overloads return response value instead of Response<T>.
+- Asynchronous calls check subscriberContext for tracing context.
+- Synchronous calls support passing tracing context in maximal overloads.
+
+## Version 1.0.0-preview.1:
+
+Version 1.0.0-preview.1 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic 
```

Comment:
Do you think it is better to put the java spec on the top as it applies to all versions.

Reply:
For now I think it makes sense here as we could have different guidelines in the future.